cmd/dtab.go

```diff
@@ -94,6 +94,80 @@ var (
 		},
 	}
 
+	dtabVersionCmd = &cobra.Command{
+		Use:     "version [name]",
+		Short:   "Get a version by name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			switch len(args) {
+			case 1:
+				ctl, err := getController()
+				if err != nil {
+					return err
+				}
+				name := args[0]
+				vd, err := ctl.Get(name)
+				if err != nil {
+					return err
+				}
+
+				fmt.Println(vd.Version)
+
+				return nil
+
+			default:
+				return errors.New("get requires a name argument")
+			}
+		},
+	}
+
+	dtabSetCmd = &cobra.Command{
+		Use:     "set [name] [prefix] [destination]",
+		Short:   "Update a dtab destination by name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			switch len(args) {
+			case 3:
+				ctl, err := getController()
+				if err != nil {
+					return err
+				}
+				name := args[0]
+				vd, err := ctl.Get(name)
+				if err != nil {
+					return err
+				}
+
+				prefix := args[1]
+				destination := args[2]
+
+				found := false
+				for _, dtab := range vd.Dtab {
+					if dtab.Prefix == "/" + prefix {
+						dtab.Destination = destination
+						found = true
+					}
+				}
```

Comment:
Perhaps rather than _set_, _replace_ and/or _append_?  I don't think we really have to target more advanced cases like the one i mention, I just want to the behavior to be clear & predictable.